transcribe/scribe_modules/nvidia_smi.py

```diff
@@ -0,0 +1,27 @@
+from . import ScribeModuleBaseClass
+from . lib.util import validate_length
+
+import sys
+import xmltodict
+
+class Nvidia_smi(ScribeModuleBaseClass):
+    
+    def __init__(self, input_dict=None, module_name=None, host_name=None,
+                 input_type=None, scribe_uuid=None):
+        ScribeModuleBaseClass.__init__(self, module_name=module_name,
+                                       input_dict=input_dict,
+                                       host_name=host_name,
+                                       input_type=input_type,
+                                       scribe_uuid=scribe_uuid)
+
+    def parse(self):
+        validate_length(len(self._input_dict['xml']), self.module)
+        self._dict["value"] = {}
+        self._dict["value"] = xmltodict.parse(self._input_dict['xml'])['nvidia_smi_log']
+        
+        if "gpu" in self._dict["value"]:
+            for gpu_keys in self._dict["value"]["gpu"]:
+                while "fan_speed" in gpu_keys:
+                    gpu_keys.popitem()
```

Comment:
I think we should flatten all arbitrary data, in this case that data is under "value.vpu". You can flatten it using the dict_to_list function available with `from . lib.util import dict_to_list`

```suggestion
self._dict["value"] = dict_to_list(self._dict["value"], "value", "gpu")
```

Reply:
@rsevilla87 `from . lib.util import dict_to_list` is producing the following error:
```cannot import name 'dict_to_list'
cannot import name 'dict_to_list'```